crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -741,6 +764,13 @@ where
             }
         };
 
+        if sec.open {
+            tokio::spawn(async move {
+                tokio::time::sleep(Duration::from_millis(200)).await;
```

Comment:
Is this delay always going to be good and enough? I guess there's no easy way we can simply wait for whatever this delay is for?

Reply:
For now, simply replacing what was removed.